docs/docs/chitchat-faqs.mdx

```diff
@@ -341,90 +77,112 @@ pipeline:
     epochs: 100
 ```
 
-Now that we've defined the message handling side, we need to make
-the dialogue handling parts aware of these changes. First, we need to add the
-new intents to our domain:
-
-```yaml-rasa title="domain.yml"
-intents:
-  - greet
-  - bye
-  - thank
-  - faq
-```
-
-We'll also need to add a [retrieval action](./retrieval-actions.mdx),
-which takes care of sending the response predicted from the `ResponseSelector`
-back to the user, to the list of actions. These actions always have to start
-with the `respond_` prefix:
+By default, the `ResponseSelector` will build a single retrieval model for all retrieval intents.
+If you want to retrieve responses for FAQs and chitchat separately, you can use multiple `ResponseSelector` components
+and specify the `retrieval_intent` key, for example:
```

Comment:
should this be phrased more like a recommendation? instead of saying "if you want to"